packages/database/src/migrations/20240508131806_/migration.sql

```diff
@@ -0,0 +1,19 @@
+BEGIN TRY
+
+BEGIN TRAN;
+
+-- AlterTable
+ALTER TABLE [dbo].[AppellantSubmission] ALTER COLUMN [siteAreaSquareMetres] FLOAT(53) NULL;
```

Comment:
Need to delete this migration and recreate with the decimal type; also be nice if the migration had a name :)

Reply:
done and done :)